### PR TITLE
rename l1/l2 pay and add chain / currency sumbols to UI

### LINKED ISSUE #120: rename "L1/L2 pay"
* although a defensible claim, state channel networks are not "an L2" in a lot of people's minds
* if we deploy onto a rollup, "L1 pay" is actually causing an L2 tx, and "L2 pay" is doing... an L3 tx?

Suggest "on chain pay" / "fast pay" or something like that.

### PR DESCRIPTION
**Pull request by geoknee**
Closes #120 

Screenshot 2023-10-19 at 10 22 11

### CODE REVIEW

File: src/Wallet.tsx [hunk 1]
```diff
@@ -280,7 +289,7 @@ const Wallet: React.FunctionComponent<{ role: Role }> = (props: {
                     });
                 }}
               >
-                <BoltIcon /> L2 Pay
+                <BoltIcon /> Bridge to {myPeerChain.symbol}
```
Comment (geoknee) [inline src/Wallet.tsx:292]: Could use the "name" field here, although that is typically a little too long to make the button look good...

We could just use "bridge" in the button, and display the chain name (and/or logo?) somewhere else on the UI?

Comment (geoknee) [reply src/Wallet.tsx:292]: i.e. Host Network / Target Network

**Comment by NiloCK** [comment]
Slightly mixed minds about this - this is a tech demo for a tech audience, but a big benefit of this system is that it abstracts away "guts" from the end user, who probably benefits most from a simple "Pay" button denominated in their local currency.

I'd lean toward putting this in, but I wonder if there's a different way to emphasize the bridging / network hopping, while also demonstrating a very friendly UI to the end user.

**Comment by geoknee** [comment]
"chain pay" / "bridge pay"?

**Comment by geoknee** [comment]
And for consistency, "host chain" ("target chain" even)

**Pull request closed without merging**